Cargo.toml

```diff
@@ -36,9 +35,8 @@ distill-loader = { version = "0.0.3", path = "loader" }
 futures = "0.3"
 serde = "1"
 uuid = "0.8.2"
-tokio = { version = "1.2", features = ["sync"] }
 serial_test = "0.5.1"
-
+async-oneshot =  "0.5.0"
```

Comment:
Get rid of dependency on `async-oneshot` -- maybe just try a bounded channel with a size of 1?

Reply:
Yeah that'd be nice!

Reply:
Removed.